### PR TITLE
March 2024 live update summary

### PR DESCRIPTION
**Pull request by kpodemski**
Summary of yesterday's live update.

### CODE REVIEW

File: content/news/public-oss-demo/2024/live-update-march-2024.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community, live-update]
+---
+
+In our third Live Update this year, we reviewed the already implemented compatibility of PrestaShop with Symfony Framework 6.4, focusing on its impact and the backward incompatible changes it introduces.
+
+## PrestaShop Live Update - March 2024
+
+In our third Live Update of the year, [Morgan Pichat](https://github.com/M0rgan01), one of the core developers at PrestaShop SA, delivered a presentation showing some of the details and challenges regarding the integration of Symfony 6.4 into PrestaShop 9.
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-march-2024.md]:
```suggestion
In this Live Update, [Morgan Pichat](https://github.com/M0rgan01), one of the core developers at PrestaShop SA, delivered a presentation showing some of the details and challenges regarding the integration of Symfony 6.4 into PrestaShop 9.
```

File: content/news/public-oss-demo/2024/live-update-march-2024.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Morgan highlighted some of the significant changes for solution authors in the PrestaShop ecosystem following the upgrade and the challenges overcome during this project. Congratulations to Morgan and the rest of the core developers team for their exceptional work on this project 🎉👏.
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-march-2024.md]:
```suggestion
Morgan highlighted some of the significant changes for solution authors in the PrestaShop ecosystem following the upgrade and the challenges overcome during this project. Congratulations to Morgan and the rest of the core developer team for their exceptional work on this project 🎉👏.
```

File: content/news/public-oss-demo/2024/live-update-march-2024.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+We will document all these changes in our developer documentation, providing a comprehensive resource for our developer community. This update is part of a series where we show impactful changes coming to PrestaShop 9.
+
+## Friends of Presta Day 2024
+
+During the Live Update, we also announced the Friends of Presta Day 2024, a two-day event held on April 2nd, 2024, in Marseille. FOP Days, organized by the Friends of Presta association, is an excellent opportunity to meet and share experiences with other community members in France. To learn more about the event, visit the [Friends of Presta website](https://friendsofpresta.org/friends-of-presta-day-fop-day/).
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-march-2024.md]:
```suggestion
During the Live Update, we also announced the Friends of Presta Days 2024, a two-day event held on April 2nd, 2024, in Marseille, France. The FOP Days, organized by the Friends of Presta association, is an excellent opportunity to meet and share experiences with other community members in France. To learn more about the event, visit the [Friends of Presta website](https://friendsofpresta.org/friends-of-presta-day-fop-day/).
```

File: content/news/public-oss-demo/2024/live-update-march-2024.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+## Q&A session
+
+We also had a Q&A session where we answered questions from the community. Remember that you can ask your questions during the presentation or beforehand.
+**If you have questions related to the open source project**, feel free to let us know in advance [using this form](https://forms.gle/FWazuZnXBtFPauFZ7). We will do our best to answer them during the Q&A session.
+
+## The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=097fUWgPuWo) Beginning of the session: the introduction
+- [05:16](https://youtu.be/097fUWgPuWo?t=316) Presentation: PrestaShop 9. Major version, major changes, by [Krystian Podemski](https://github.com/kpodemski)
+- [10:06](https://youtu.be/097fUWgPuWo?t=606) Presentation: Migration to Symfony 6.4 in PrestaShop 9, by [Morgan Pichat](https://github.com/M0rgan01)
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-march-2024.md]:
```suggestion
- [10:06](https://youtu.be/097fUWgPuWo?t=606) Presentation: migration to Symfony 6.4 in PrestaShop 9, by [Morgan Pichat](https://github.com/M0rgan01)
```

File: content/news/public-oss-demo/2024/live-update-march-2024.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+- [19:37](https://youtu.be/097fUWgPuWo?t=1177) Q&A
+
+You can use the timestamps to jump directly to the parts you want to watch.
+
+{{< cta "https://www.youtube.com/watch?v=097fUWgPuWo" "info" >}}Watch replay on YouTube{{< /cta >}}
+
+Below are links to projects mentioned during this live update:
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-march-2024.md]:
```suggestion
Below are links to projects mentioned during this Live Update:
```

File: content/news/public-oss-demo/2024/live-update-march-2024.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+- [FOP Days 2024](https://friendsofpresta.org/friends-of-presta-day-fop-day/)
+- [Symfony 6.4 integration to PrestaShop core](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+6.4)
+
+Also, you can take a look at the slides presented during the demo [here](https://docs.google.com/presentation/d/1plGHxQ7vNRxgPQsew3aZEB0b0CGfUUkLr2E1_FRSv9o/edit?usp=sharing).
+
+We are available for feedback about the project and the software. We also invite everyone to join us on [Slack](https://www.prestashop-project.org/slack/).
+
+## Next live update in April
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-march-2024.md]:
```suggestion
## Next Live Update in April
```

File: content/news/public-oss-demo/2024/live-update-march-2024.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+We will be back in April with another live update. Mark your calendars for April 10th, 2024, at [4 pm (Paris time)](https://time.is/1600_10_Apr_2024_in_Paris). We will share more details about the agenda soon.
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-march-2024.md]:
```suggestion
We will be back in April with another Live Update. Mark your calendars for April 10th, 2024, at [4 pm (Paris time)](https://time.is/1600_10_Apr_2024_in_Paris). We will share more details about the agenda soon.
```

**Review by coffrep — APPROVED** [review]
Hi @kpodemski thanks for the great Live Update summary post, I've added a few suggestions and approved on my end with edits. And yes there is a Marseilles in Illinois, USA ;)

**Comment by kpodemski** [comment]
thanks @coffrep 👍🏻

**Pull request merged**